examples/nlp/gpt/conf/gpt_ppo_actor.yaml

```
@@ -9,7 +9,13 @@ trainer:
   precision: bf16
 
   ppo:
-    critic_warmup_steps: 1
+    # How many steps we train warmup the critic for(without training the policy)
```

That missing space... :p
```suggestion
    # How many steps we train warmup the critic for (without training the policy)
```

---

sorry i keep forgetting!